frontend/src/app/app.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="container" style="max-width: 550px">
+    <div *ngIf="!isLoggedIn">
+        <div>
+            <label> Login with Google</label>
+            <asl-google-signin-button type="standard" size="large" text="signin_with"></asl-google-signin-button>
+        </div>
+    </div>
+    <div *ngIf="isLoggedIn">
+        <div class="form-group">
+            <label>First Name</label>
+            <input type="text" class="form-control" [value]="socialUser.firstName" id="firstname" readonly />
+        </div>
+        <div class="form-group">
+            <label>Last Name</label>
+            <input type="text" class="form-control" [value]="socialUser.lastName" id="lastname" readonly />
+        </div>
+        <div class="form-group">
+            <label>Email</label>
+            <input type="text" class="form-control" [value]="socialUser.email" id="email" readonly />
+        </div>
+        <div>
+            <p>idToken: {{ socialUser.idToken }}</p>
+        </div>
+        <button type="button" (click)="logOut()" class="btn btn-primary">Log Out</button>
+    </div>
+</div>
+
+<app-loading-spinner></app-loading-spinner><router-outlet></router-outlet>
```

Comment:
Why is it in app? And why you created all these fields here?

Reply:
it will migrated to LoginPage as a separate module, here it was created for testing

Reply:
Then, please update it and merge this PR when Login component will be ready

Reply:
Or at least move all these parts to auth module